### DISCUSSION TITLE
Websockets not accessible

### DISCUSSION THREAD

**Discussion by markfaine** [nginx-proxy/nginx-proxy#1670] [Q&A]
If you have a question, DO NOT SUBMIT a new issue.  Please ask the question on the Q&A Group: https://groups.google.com/forum/#!forum/nginx-proxy **<-- Does not exist**

I'm seeing the following in the console with the application:

```Firefox can’t establish a connection to the server at wss://semaphore.nisn.nasa.gov/api/ws.```

I saw one other issue #128 regarding WSS but it wasn't helpful.

According to the nginx [documentation ](http://nginx.org/en/docs/http/websocket.html)there needs to be a Connection header change:

I have configured a custom vhost config for the endpoint:

```
...
  location /api/ws {
        proxy_pass http://semaphore.nisn.nasa.gov/api/ws;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
    }
...
```

However, this doesn't seem to help.  What would you suggest?
 
[semaphore.nisn.nasa.gov.txt](https://github.com/nginx-proxy/nginx-proxy/files/5373120/semaphore.nisn.nasa.gov.txt)

Nothing useful in the docker logs for the nginx-proxy container.

**Comment by elhe26** [comment]
Any updates?

@markfaine were you able to solve your issue?
